calyx-utils/src/position.rs

```diff
@@ -184,6 +184,17 @@ impl GPosIdx {
         (buf, out_idx, out_line)
     }
 
+    /// returns:
+    /// 1. the name of the file the span is in
+    /// 2. the starting line of a span
+    pub fn get_line_num(&self) -> (&String, usize) {
+        let table = GlobalPositionTable::as_ref();
+        let pos_data = table.get_pos(self.0);
+        let file_name = &table.get_file_data(pos_data.file).name;
+        let tup = self.get_lines();
```

Comment:
nit: for clarity here it would be better to deconstruct the tuple with pattern
matching rather than grabbing the field. Ex:
```rust
let (_, _, lineno) = self.get_lines();
```
This makes it a bit easier to understand what's going on in the code